### PR TITLE
Ember cli classic – merge take 2

### PR DESCRIPTION
**Pull request by amiller-gh**
I screwed up pulling in https://github.com/linkedin/css-blocks/pull/185... it was pointing at a branch that wasn't master ¯\_(ツ)_/¯ 

Lets do this again.

### CODE REVIEW

**Comment by bartocc** [comment]
Now that it is merged in master, can we use css-blocks in an ember.js app ?

**Comment by amiller-gh** [comment]
Because its still pre-1.0 feel free to use it, but at your own risk 🙂Would definitely appreciate some help kicking the wheels a bit and seeing what breaks.

Give the addon's readme a read and see if, even with the caveats, it will work for your use cases right now. I expect most of those six outstanding items to be finished in the next few months. I'll have RFC issues for all six up this week. 

https://github.com/linkedin/css-blocks/blob/fb7a54076270bdb4102b2931374ff12803f1d348/packages/%40css-blocks/ember-cli/README.md

Its also not published yet! Waiting on @chriseppstein to cut a beta version of the addon.

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,10 @@ language: node_js
 
 node_js:
   - "8"
+<<<<<<< HEAD
```
Comment (dbbk) [inline .travis.yml]: Looks like this git conflict accidentally made it in!

Comment (amiller-gh) [reply .travis.yml]: 😫Lets pretend this never happened...

**Pull request merged**